generators/app/templates/client/package.json

```diff
@@ -21,25 +25,37 @@
     "react-dom": "^15.2.1",
     "react-redux": "^4.4.5",
     "react-router": "^2.6.0",
+    "react-router-redux": "^4.0.5",
     "redux": "^3.5.2",
+    "redux-saga": "^0.11.1",
     "sass-loader": "^4.0.0",
     "url-loader": "^0.5.7",
     "webpack": "^1.13.1",
     "webpack-merge": "^0.14.0",
     "whatwg-fetch": "^1.0.0",
-    "yaml-loader": "^0.4.0"
+    "yaml-loader": "^0.4.0",
+    "dotenv": "^2.0.0",
+    "uuid": "^2.0.2",
+    "raw-loader": "^0.5.1",
+    "classnames": "^2.2.5",
+    "immutable": "^3.8.1",
+    "postcss": "^5.1.2",
+    "postcss-cssnext": "^2.7.0",
+    "postcss-reporter": "^1.4.1",
+    "postcss-loader": "^0.11.1"
   },
   "devDependencies": {
     "foreman": "^1.4.1",
-    "react-hot-loader": "^3.0.0-beta.2",
-    "style-loader": "^0.13.1",
     "webpack-dev-server": "^1.14.1",
     "yo": "^1.8.4",
-    "generator-prelude": "<%= generatorVersion %>"
+    "generator-prelude": "<%= generatorVersion %>",
+    "enzyme": "^2.4.1",
+    "mocha": "^3.0.2"
   },
   "scripts": {
-    "heroku-postbuild": "webpack --config config/webpack.js",
-    "setup": "npm install --only=dev && yo prelude:setup",
-    "start": "nf start -j Procfile.dev"
+    "heroku-postbuild": "webpack --config config/production.js",
+    "start": "node server.js & webpack-dev-server --progress --colors --config config/webpack.development.js",
```

Comment:
This will make the server.js go into background, which makes it kinda hard to kill it since `Ctrl C` will only reach the webpack-dev-server command. That's one of the reasons I made the webpack-dev-server go into the server.js, so we won't need to keep adding stuff here, and also because in server.js we have the whole node API on our side.